datagrepper/app.py

```diff
@@ -381,11 +350,11 @@ def raw():
 
         # Convert our messages from sqlalchemy objects to json-like dicts
         messages = [msg.__json__() for msg in messages]
-        if grouped:
+        if grouped:  # pragma: no cover
             messages = fedmsg.meta.conglomerate(messages, **fedmsg_config)
             for message in messages:
                 message["date"] = arrow.get(message["timestamp"]).humanize()
-        elif meta:
+        elif meta:  # pragma: no cover
```

Comment:
This one is a different beast, we want to use the message's schema properties instead of fedmsg's `msg2*` functions. It should probably be the focus of a different PR, but in the meantime we shouldn't exclude it from coverage.